packages/@react-spectrum/searchfield/src/SearchField.tsx

```diff
@@ -63,7 +63,7 @@ function SearchField(props: SpectrumSearchFieldProps, ref: RefObject<TextFieldRe
     <TextFieldBase
       {...otherProps}
       labelProps={labelProps}
-      inputProps={inputProps}
+      inputProps={{...inputProps, defaultValue: undefined}}
```

Comment:
This fails quite spectacularly with Preact: https://codesandbox.io/s/preact-input-defaultvalue-twice-p0l25 (try to remote the text using backspace).

So `useTextField` or `useSearchField`? Probably `useSearchField` because the default value is handled in `useSearchFieldState`?

I can add a "unit test" for the hook's return value.

Reply:
hahaha, that's amazing
lets see if @devongovett has a preference for where to handle it

Reply:
Done: 3b12657